libs/ui/editor/src/lib/blocks/teaserListBlock.tsx

```diff
@@ -0,0 +1,107 @@
+import {useEffect, useState} from 'react'
+import {useTranslation} from 'react-i18next'
+import {MdEdit} from 'react-icons/md'
+import {Drawer, IconButton as RSIconButton, Panel} from 'rsuite'
+
+import {BlockProps} from '../atoms/blockList'
+import {PlaceholderInput} from '../atoms/placeholderInput'
+import {TeaserListBlockValue} from './types'
+import styled from '@emotion/styled'
+import {ContentForTeaser} from './teaserGridBlock'
+import {TeaserListConfigPanel, useTeaserTypeText} from '../panel/teaserListConfigPanel'
+
+const TeaserGrid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 24px;
+  margin-top: 24px;
+`
+
+const TeaserWrapper = styled('article')`
+  position: relative;
+  min-height: 250px;
+  background-color: #f7f9fa;
+`
+
+const PreviewPanel = styled(Panel)`
+  overflow: hidden;
+  background-color: #f7f9fa;
+  display: grid;
+  position: relative;
+`
+
+const Count = styled.p`
+  text-align: center;
+`
+
+const IconButton = styled(RSIconButton)`
+  position: absolute;
+  right: 0;
+`
+
+const InfoList = styled.ul`
+  list-style: none;
+  padding: 12px;
+`
+
+export const TeaserListBlock = ({value, onChange, autofocus}: BlockProps<TeaserListBlockValue>) => {
+  const {filter, teasers, skip, take, teaserType} = value
+  const [isDialogOpen, setIsDialogOpen] = useState(false)
+  const {t} = useTranslation()
+  const teaserTypeText = useTeaserTypeText()
+
+  const isEmpty = !teasers.length
+  const teasersToDisplay = teasers.slice(0, 6)
```

Comment:
should this value be hardcoded? 6 I mean

Reply:
Yeah it's just how much to preview